pkg/api/install/tags_test.go

```diff
@@ -95,6 +95,9 @@ func checkInternalJsonTags(objType reflect.Type, seen *map[reflect.Type]bool, t
 	if internalTypesWithAllowedJsonTags.Has(objType.Name()) {
 		return
 	}
+	if objType.Kind() != reflect.Struct {
```

Comment:
@smarterclayton was just going to ask if this is a 'safe' change... having types that are lists seems reasonable to me.